source/javascripts/controllers/_StackController.js.erb

```diff
@@ -36,11 +36,18 @@
 					}
 					return machineType.stackType === stack.type;
 				});
-				result[stack.id] = _.groupBy(stackMachineTypes, 'tag');
+				result[stack.id] = _.groupBy(stackMachineTypes, 'chip');
 
 				return result;
 			}, {});
 
+			viewModel.getStackMachineTypeChipOrder = function(stackId) {
+				if (viewModel.stackMachineMap) {
+					return Object.keys(viewModel.stackMachineMap[stackId] || {}).sort();
+				}
+				return [];
+			}
+
```

Comment:
Here we're basically saying "sort the groups alphabetically by the value of `chip`", right? i.e. relying on the fact that `apple` comes before `intel`.

So, if for example if we started supporting AWS's Graviton chips (and used either `aws` or `graviton` as the value for `chip`), those would insert themselves between Apple and Intel. 

In less of a rush, I'd wish for something more robust -- but don't change it now. 🙂

Reply:
So I thought about this and decided that doing a plain sort is the simplest option for now that will allow us to group by the value of `chip` without knowing what the possible values are. I think it would be better for us to add a new machine `chip` value and see the machine groups out of order than to not see the new machine. also this protects us in the unlikely event that `apple` is changed to `Apple` or something like that.